src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.cpp

```diff
@@ -105,15 +105,27 @@ bool Instance::HasFeature(MicrowaveOvenControl::Feature feature) const
     return mFeature.Has(feature);
 }
 
-uint8_t Instance::GetCountOfSupportedWattLevels() const
+bool Instance::CheckSupportedWattLevelsValid() const
 {
     uint8_t wattIndex = 0;
     uint16_t watt     = 0;
     while (mDelegate->GetWattSettingByIndex(wattIndex, watt) == CHIP_NO_ERROR)
     {
         wattIndex++;
     }
-    return wattIndex;
+    return wattIndex > 0 && watt > 0;
```

Comment:
Why are we now adding the check that `watt > 0`?

Reply:
just in case the SDK user set watt number to 0.